### ISSUE TITLE
Book DAC test and create prototype for new features added since last audit for census

### ISSUE THREAD

**Issue by yatesn** [ONSdigital/design-team#10]
Features needing an accessibility audit include:

## Foundations
- [x] Links - **DS and eQ**
  - [x] [New hover style](https://ons-design-system.netlify.app/foundations/typography/links/examples/link/) added so state didn't rely on colour change alone – thicker underline – is the change in thickness significant enough?
  - [x] External link missing screen reader hidden notice (that opens in a new tab)
- [x] Highlighting - **DS only**
  - [x] Any issues with focus colour clashing with similar colour used for [highlighting important information](https://ons-design-system.netlify.app/foundations/typography/#highlighting)?
  - [x] Is `<em>` semantically the correct tag for highlighting piped answers etc?
  - [x] Should the font-weight of an `<em>` be bold or left to style as italics?

## Components
- [x] Accordions (check [issues found with GDS's previous version](https://github.com/alphagov/govuk-design-system-backlog/issues/1#issuecomment-995675898) don't also apply to our component) - **DS only**
  - [x] Are the 'show/hide' buttons for each accordion item necessary?
  - [x] Should the title and button be combined and focusable?
  - [x] Does title link style cause issues with expectation of behaviour (are they mistaken for links to a new page)? Is the link style required to indicate it needs to be interacted with and it not a heading?
  - [x] Should we use `<details>`/`<summary>` elements instead? Any issue with multiples on same page?
  - [x]  See and compare versions:
    - [x]  [current implementation](https://ons-design-system.netlify.app/components/accordion/examples/accordion/) (link style + separate button) 
    - [x]  [refactored implementation](https://deploy-preview-2013--ons-design-system.netlify.app/components/accordion/examples/accordion/) in ONSdigital/design-system#2013 (`<details>` used, underline and button removed)
    - [x]  [simple accordion](https://ons-design-system.netlify.app/components/accordion/examples/accordion-simple/) (new variant, needs auditing)
- [x]  Buttons - **DS and eQ**
  - [x]  Do [links (`<a>`) styled as buttons](https://ons-design-system.netlify.app/components/button/examples/button-link/) with `role="button"` cause issues?
  - [x]  New [print](https://ons-design-system.netlify.app/components/button/examples/button-print/) and [download](https://ons-design-system.netlify.app/components/button/examples/button-download/) buttons - **eQ only**
- [x] Card – **DS**
  - [x] Are aria attributes required on the wrapping `div` (it's not an interactive element)?
- [x]  Checkboxes/radios - **DS and eQ**
  - [x]  Conditionally revealed inputs ([known issues with GDS component](https://design-system.service.gov.uk/components/checkboxes/#known-issues))
    - [x]  Issues caused by `aria-expanded` not always announcing when input shown/hidden
    - [x]  Confusion over use of component if the revealed content is too complex
    - [x]  [with conditionally revealed text input](https://ons-design-system.netlify.app/components/checkboxes/#checkbox-with-revealed-text-input)
    - [x]  [with conditionally revealed select](https://ons-design-system.netlify.app/components/checkboxes/#checkbox-with-revealed-drop-down)
    - [x]  [with conditionally revealed checkboxes](https://ons-design-system.netlify.app/components/checkboxes/#checkbox-with-revealed-checkboxes)
    - [x]  [with conditionally revealed radios](https://ons-design-system.netlify.app/components/checkboxes/#checkbox-with-revealed-radios)
  - [x]  "[Select all](https://ons-design-system.netlify.app/components/checkboxes/examples/checkboxes-autoselect/)" button
- [x]  Collapsible - **DS and eQ**
  - [x]  (See issues raised in 'Accordions')
- [x]  Document list (new component, needs auditing) - **DS only**
- [x]  Input - **DS**
  - [x]  [Character limit checker](https://ons-design-system.netlify.app/components/input#character-limit-checker) (new variant, needs auditing)
- [x] Header - **DS*
  - [x] Multi-level navigation
- [x]  Mutually exclusive - **DS and eQ**
  - [x]  When the [mutually exclusive checkbox](https://ons-design-system.netlify.app/components/mutually-exclusive/examples/checkboxes/) is selected, on focus of one of the options above, should we announce that selecting it will uncheck the mutually exclusive checkbox?
  - [x]  Or are all the "unselected" announcements overkill, considering the label for the mutually exclusive checkbox implies that any previously selected option won't apply?
- [ ]  Panel - **DS and eQ**
  - [x]  New visually-hidden prefix 
  - [ ]  [Warning panel with timeout](https://ons-design-system.netlify.app/components/timeout-panel/examples/warn/) (new variant, needs auditing)
- [x]  Search - **DS only**
  - [x]  with [placeholder](https://ons-design-system.netlify.app/components/search/examples/search-placeholder/)
  - [x]  with [character check](https://ons-design-system.netlify.app/components/search/examples/search-character-check/)
- [x]  Section navigation - **DS only**
  - [x]  [Vertical](https://ons-design-system.netlify.app/components/section-navigation/examples/section-navigation-vertical/) (new variant, needs auditing)
- [x]  Select - **DS only**
  - [x]  with [inline label](https://ons-design-system.netlify.app/components/select/examples/select-inline/) (new variant, needs auditing)
- [x]  Share page - **DS only**
  - [x]  [new component](https://ons-design-system.netlify.app/components/share-page/examples/share-page/), needs auditing
- [x]  Table of contents - **DS only**
  - [x]  [skip over table of contents link](https://ons-design-system.netlify.app/patterns/guide/examples/guide-overview/) (new enhancement, needs validating)  
  - [x]  In-page navigation](https://ons-design-system.netlify.app/components/table-of-contents/examples/table-of-contents-sticky/) (new variant, needs auditing)
    - [x]  Does this variant also need a skip link? The assumption is not, due to it not persisting on other pages.
- [x]  Tabs - **DS only**
  - [x]  Any issues with use of the ARIA `tablist` role (as [mentioned here](http://simplyaccessible.com/article/danger-aria-tabs/#acc-heading-2))?
  - [x]  Should the `tabpanel` be part of the tab order?
  - [x]  Should each tab be tabble?
- [x]  Table - **DS only**
  - [x]  Sortable table header cells are s `<button>` styled as a link – does this cause issues?
- [x]  Textarea - **DS and eQ**
  - [x]  with [character counter](https://ons-design-system.netlify.app/components/textarea/examples/textarea-character-limit/)
    - [x]  The `aria-describedby` references the character limit instead of the hint text (like other inputs) – how should this be handled?
    - [x]  Voiceover for iOS interrupts the character counter after entering characters

## Patterns
- [x]  Dates and Durations - **DS only**
  - [x]  [Fieldset of inputs with suffixed labels](https://ons-design-system.netlify.app/components/duration/examples/duration-address/) - the autofill "candidate list" announcement interrupts the reading of the suffix labels on iOS Voiceover
- [x]  Numeric values - **DS only**
  - [x]  Inputs with [prefixed/suffixed units and hint text](https://ons-design-system.netlify.app/components/input/examples/suffixed/)
    - [x]  On Voiceover for iOS, the hint text appears to interrupt the announcement of units provided by the `title` attribute as it reads the `aria-describedby` by instead.
- [ ]  Errors - **eQ**
  - [ ]  When you select a link in the error summary panel at the top of the page, we didn't put focus on the first option in a radio/checkbox fieldset – would be good to clarify if this is acceptable.
  - [ ]  Is it an issue that the error summary header is an `<h2>` yet placed above the page heading `<h1>`?
  - [ ]  When page loads with an error, should the error summary panel focus state be visible? Should `tabindex=-1` be removed after focus is lost?
  - [ ]  Check [prototype](https://ons-design-system.netlify.app/patterns/correct-errors/examples/errors-proto/)
- [x]  Download resources - **DS only**
  - [x]  [new pattern](https://ons-design-system.netlify.app/patterns/download-resources/examples/download-resources/), needs auditing
- [ ]  Extend a session - **eQ**
  - [ ]  [new pattern](https://ons-design-system.netlify.app/patterns/extend-session/examples/timeout/), needs auditing
- [ ]  Send and reply to messages - **RASRM**
  - [ ]  [new pattern](https://ons-design-system.netlify.app/patterns/send-and-reply-to-messages/examples/conversation/), needs auditing
- [x] News - **DS only**
  - [x]  [new pattern](https://ons-design-system.netlify.app/patterns/news/), needs auditing
- [x]  Question - **DS and eQ**
  - [x]  [Legend as question title](https://ons-design-system.netlify.app/components/radios/examples/radios-other/) (wraps page heading in a `<legend>`when question and fieldset answer is the only thing on the page)
- [x]  DS site search - **DS**
  - [x]  'Search the design system' persistent across [the website](https://ons-design-system.netlify.app/)
